### DISCUSSION TITLE
How to configure HelpTextStyle.Basic

### DISCUSSION THREAD

**Discussion by Nutr1k** [bilal-fazlani/commanddotnet#433] [Q&A] [answered]
```
static int Main(string[] args) =>
            new AppRunner<Program>().Configure(x=>x.CustomHelpProvider=new Settings(x.AppSettings)).Run(args);
```

Did I predetermine correctly?

```
class Settings : HelpTextProvider
    {
        public Settings(AppSettings appSettings, string appName = null) : base(appSettings, appName)
        {
            appSettings.Help.TextStyle = HelpTextStyle.Basic;
        }
    }
```

It just doesn't work

**Comment by drewburlingame** [comment, marked as answer]
Yeah, what you're doing doesn't really make sense.  

If you want to use the Basic text provider, you can do 

`new AppRunner<Program>(new AppSettings{Help = {TextStyle = HelpTextStyle.Basic}}).Run(args)`.  

There's an example in the [file](https://github.com/bilal-fazlani/commanddotnet/blob/master/CommandDotNet.DocExamples/GettingStarted/Getting_Started_300_Help.cs#L78) that tests and generates our help examples.
